src/Tizen.NUI.Components/Controls/Navigation/Navigator.cs

```diff
@@ -24,27 +24,40 @@ namespace Tizen.NUI.Components
 {
     /// <summary>
     /// The Navigator is a class which navigates pages with stack methods such as Push and Pop.
-    ///
+    /// </summary>
+    /// <remarks>
     /// With Transition class, Navigator supports smooth transition of View pair between two Pages
-    /// by using PushWithTransition(Page) and PopWithTransition() methods.
-    /// If there is View pair of current top Page and next top Page those have same View.TransitionOptions.TransitionTag,
+    /// by using <see cref="PushWithTransition(Page)"/> and <see cref="PopWithTransition()"/> methods.
+    /// If current top Page and next top Page have same <see cref="View.TransitionOptions.TransitionTag"/>,
```

Comment:
It turns out, that writing like this will not work. The docfx allows (class full name)(.optional class member) and does not follow through other members.
```suggestion
    /// If current top Page and next top Page have same <see cref="View"/>.<see cref="View.TransitionOptions"/>.<see cref="TransitionOptions.TransitionTag"/>,
```

Reply:
Thank you. I fixed the sentence.